doc/usecases/polyculture/review_plant_relationships.md

```diff
@@ -0,0 +1,28 @@
+# Use Case: Review Plant Relationships
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User, App User
+- **Brief:** The quality of a user created plant relationship is assessed by others.
+- **Status:** Assigned
+- **Assignee:** Benjamin
+
+## Scenarios
+
+- **Precondition:**
+  - The user is logged into the app.
+  - She is looking at a [plant relationship created by another user](add_plant_relationships.md).
+- **Main success scenario:**
+  - The user comments on the relationship to note something that seems relevant to her.
+  - The user approves the relationship because she agrees with it.
+- **Alternative scenario:**
+  - The user only comments on the relationship but doesn't approve it and might suggest a different relationship (e.g. neutral). 
+  - No changes are made to the relationships confidence score.
+  - One administrator decides on the outcome.
+- **Error scenario:**
+- **Postcondition:**
```

Comment:
How about informing users who reviewed the relationship about the outcome?

Reply:
Do we plan on doing some notification system within the app or emails, etc.?
@markus2330

Reply:
We will have a notification system via Nextcloud and at login ( #223). I think it is a good idea to add reviewed plant relationships also to #223 (if you have the plant in your favorites). But having a Nextcloud chat for debated relationships also sounds like a good idea. I would leave this open for now and add here:

    - On changes of the confidence score, [users] get notified about the updated score.

The word [users] needs to be fixed with the users that proposed changes.